earthly/docs/Earthfile

```diff
@@ -32,17 +32,13 @@ deps:
         tiff-dev \
         tk-dev \
         tcl-dev  \
-        git \
-        make
+        git
```

Comment:
I don't think this file should be changed

Reply:
this is the original file when I've just branched out my PR
right now I'm trying to back track to why the CI is erroring